pytorch_to_returnn/torch/nn/modules/operator.py

```diff
@@ -31,18 +31,16 @@ def create_returnn_layer_dict(self, limit, start, delta, dtype, sparse=False) ->
   def _get_output_shape_from_returnn(self,
                                      inputs_flat: List[Optional[Union[Tensor, int, bool]]], layer: LayerBase
                                      ) -> Tuple[Tuple[int, ...], Dict[int, int]]:
+    from .shape import SizeValue
     limit, start, delta, *_ = inputs_flat
-    limit_dim = None
     if isinstance(limit, Tensor):
       assert limit.is_defined
-      limit_dim = None
+      size = SizeValue((int(limit.numpy()) - int(start)) // int(delta))
       if limit.returnn_naming_entry.is_size_value is not None:
-        limit_dim = limit.returnn_naming_entry.is_size_value.dim_tag
-      limit = limit.numpy()
-    from .shape import SizeValue
-    size = SizeValue((int(limit) - int(start)) // int(delta))
-    if limit_dim:
-      size.dim_tag = (limit_dim - int(start)) // int(delta)
```

Comment:
Why did you change the old code so much?
Wouldn't it have been enough to just add `size.originating_tensor = ...` here?

Reply:
I found this 
```
limit_dim = None
...
if limit_dim:
...
```
unnecessarily long and harder to read, especially if we do the same for `originating_tensor` now. But if you prefer, I can also add `originating_tensor` in the same way.

Reply:
Why so complicated? Instead of `limit_dim`, you could also directly keep the `size_value`.